docs/about-ensembl/accessing-data/apis/refget.md

```diff
@@ -7,21 +7,36 @@ description: Information about Ensembl's Refget service
 
 ## What is refget?
 
-Refget is a Global Alliance for Genomics and Health (GA4GH) standard which enables access to reference genomic sequences using a checksum to unambiguously identify data.  More information about the standard can be found on the [Refget API Specification site](https://samtools.github.io/hts-specs/refget.html), but here is an example of its usage with Curl:
+Refget is a Global Alliance for Genomics and Health (GA4GH) standard which enables access to reference genomic sequences using a checksum to unambiguously identify data. More information about the standard can be found on the [Refget API Specification site](https://samtools.github.io/hts-specs/refget.html), but here is an example of its usage with curl:
 
+```bash
+curl -H'Accept: text/plain' "https://beta.ensembl.org/data/refget/sequence/17dab79b963ccd8e7377cef59a54fe1c?start=32315085&end=32315185"
 
+AAGCTTTTGTAAGATCGGCTCGCTTTGGGGAACAGGTCTTGAGAGAACATCCCTTTTAAGGTCAGAACAAAGGTATTTCATAGGTCCCAGGTCGTGTCCC%
 ```
-curl -H'Accept: text/plain' "https://beta.ensembl.org/data/refget/sequence/17dab79b963ccd8e7377cef59a54fe1c?start=32315086&end=32315186"
 
-AGCTTTTGTAAGATCGGCTCGCTTTGGGGAACAGGTCTTGAGAGAACATCCCTTTTAAGGTCAGAACAAAGGTATTTCATAGGTCCCAGGTCGTGTCCCG%      
+In the example above, the reference sequence identified by the checksum `17dab79b963ccd8e7377cef59a54fe1c` and the data between the genomic coordinates 32315086 and 32315185 (100bp) is returned.
```

Comment:
> and the data between the genomic coordinates 32315086 and 32315185 (100bp) is returned

Do you want to specify here that this is in 1-based coordinates? Because the url says `?start=32315085&end=32315185`, being in refget's coordinates. I know that you talk about the coordinate systems more below.

Reply:
updated